js/stdlib/ts/ALGO.ts

```diff
@@ -2341,21 +2343,36 @@ const balanceOfM = async (acc: Account, token: Token | null): Promise<BigNumber
         return false;
       }
     } else {
-      // Indexer token balance
-      const tokenId = bigNumberToNumber(token);
-      const client = await getAlgodClient();
-      const query = client.accountAssetInformation(addr, tokenId) as ApiCall<AccountAssetInfo>;
-      const accountAssetInfoM = await doQueryM_(dhead, query);
-      if ('val' in accountAssetInfoM) {
-        const assetHolding = accountAssetInfoM['val']['asset-holding'];
-        return assetHolding ? bigNumberify(assetHolding['amount']) : false;
-      } else {
-        return false;
-      }
+      return indexerBalanceOfMNonNet(addr, token, dhead);
```

Comment:
This function was labeled "indexer token balance", but it is actually the node balance and the code is identical